Sources/MisticaSwiftUI/Components/Snackbar/View+Snackbar.swift

```diff
@@ -9,23 +9,44 @@
 import Foundation
 import SwiftUI
 
+public enum CroutonDismissInterval {
+    case fiveSeconds
+    case tenSeconds
+    case infinite
+
+    var timeInterval: TimeInterval? {
+        switch self {
+        case .fiveSeconds:
+            return 5
+        case .tenSeconds:
+            return 10
+        case .infinite:
+            return nil
+        }
+    }
+}
```

Comment:
Maybe a more generic approach to ease adding more intervals in the future?

```suggestion
public enum CroutonDismissInterval {
    case seconds(TimeInterval)
    case infinite

    var timeInterval: TimeInterval? {
        switch self {
        case .seconds(let value):
            return value
        case .infinite:
            return nil
        }
    }
}
```

Reply:
We can't do this because you let the user to add any interval and its definition does not allow this.